docs/user_documentation.md

```diff
@@ -1434,7 +1433,7 @@ Trace automatically extracts metadata information for Microsoft Office 365 Data
 | Calculated               | Family Group                  | Fixed-Length Text | Group the file belongs to (used to identify the group if attachment fields are not used). Formerly "Group Identifier" |
 | Calculated               | File Extension                | Fixed-Length Text | The extension of the file, as assigned by the processing engine after it reads the header information from the original file |
 | Calculated               | File Name                     | Fixed-Length Text | The original name of the file                                |
-| DEPRECATED / DO NOT USE  | ~~File Size~~                 | Decimal           | This field is automatically provided by Relativity on import. **DO NOT MAP** this field in integration point profile. This field will be removed from Trace Document Extraction Field in the future release of Trace |
+| Calculated               | File Size                     | Decimal           | This field is automatically provided by Relativity on import. However, this field can also be calculated and set by Trace. Trace calculates this field as the size of the native file. **IMPORTANT:** Since both Relativity and Trace can provide a value for this field, it is up to user to decide how to use this field and decide which value to store.                                                      |
```

Comment:
As a user, I'm confused by what action I need to take here. How do I decide whether Relativity populates it or Trace populates it? What is the difference between Relativity and Trace? What is the difference between the two values?

Reply:
@wfreltrace I spoke with Misha on this one. We want to take stronger stance on the action our users should take. Can we go with "**DO NOT MAP** this field in integration point profile. A value for this field is automatically calculated by Relativity on import."

Reply:
@peterhallerREL So change the description to what you've written but leave the "calculated" and "file size" as is in this change?